tests/foreman/cli/test_errata.py

```diff
@@ -1390,9 +1448,79 @@ def test_positive_user_permission(self):
         @Assert: Check that the new user is able to see errata for one
         product only.
 
-        @caseautomation: notautomated
-
         """
+        user_password = gen_alphanumeric()
```

Comment:
yes you are right @svtkachenko using gen_string would more be suitable as used everywhere